docs/csharp/programming-guide/classes-and-structs/index.md

```diff
@@ -11,101 +11,99 @@ helpviewer_keywords:
   
 ## Interfaces  
 
- Classes and structs can inherit multiple interfaces. To inherit from an interface means that the type implements all the methods defined in the interface. For more information, see [Interfaces](../interfaces/index.md).  
+Classes and structs can inherit multiple interfaces. To inherit from an interface means that the type implements all the methods defined in the interface. For more information, see [Interfaces](../interfaces/index.md).  
   
 ## Generic Types  
 
  Classes and structs can be defined with one or more type parameters. Client code supplies the type when it creates an instance of the type. For example The <xref:System.Collections.Generic.List%601> class in the <xref:System.Collections.Generic> namespace is defined with one type parameter. Client code creates an instance of a `List<string>` or `List<int>` to specify the type that the list will hold. For more information, see [Generics](../generics/index.md).  
   
 ## Static Types  
 
- Classes (but not structs) can be declared as [static](../../language-reference/keywords/static.md). A static class can contain only static members and cannot be instantiated with the new keyword. One copy of the class is loaded into memory when the program loads, and its members are accessed through the class name. Both classes and structs can contain static members. For more information, see [Static Classes and Static Class Members](./static-classes-and-static-class-members.md).  
+Classes (but not structs) can be declared as [static](../../language-reference/keywords/static.md). A static class can contain only static members and can't be instantiated with the `new` keyword. One copy of the class is loaded into memory when the program loads, and its members are accessed through the class name. Both classes and structs can contain static members. For more information, see [Static classes and static class members](./static-classes-and-static-class-members.md).
   
-## Nested Types  
+## Nested Types
 
- A class or struct can be nested within another class or struct. For more information, see [Nested Types](./nested-types.md).  
+A class or struct can be nested within another class or struct. For more information, see [Nested Types](./nested-types.md).
   
 ## Partial Types  
 
- You can define part of a class, struct or method in one code file and another part in a separate code file. For more information, see [Partial Classes and Methods](./partial-classes-and-methods.md).  
+You can define part of a class, struct or method in one code file and another part in a separate code file. For more information, see [Partial Classes and methods](./partial-classes-and-methods.md).
   
 ## Object Initializers  
```

Comment:
Should mention `with` initializers as well? or at least some link?

Reply:
I wouldn't because `with` initializers only apply to types that conform to a pattern.